beacon_node/network/src/sync/sampling.rs

```diff
@@ -311,34 +367,52 @@ impl<T: BeaconChainTypes> ActiveSamplingRequest<T> {
         // Progress requests
         // If request fails retry or expand search
         // If all good return
-        let Some(request) = self.column_requests.get_mut(&column_index) else {
-            warn!(
-                self.log,
-                "Received sampling response for unrequested column index"
-            );
+        let Some(column_indexes) = self
```

Comment:
The comment above is probably outdated now and can be removed

Reply:
Removed in https://github.com/sigp/lighthouse/pull/6256/commits/acedd7a0a9b5610a85bad71f1aed052f2441998b